google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerOptions.java

```diff
@@ -1081,6 +1085,25 @@ public Builder setCompressorName(@Nullable String compressorName) {
       return this;
     }
 
+    /**
+     * Sets the {@link DirectedReadOptions} that specify which replicas or regions should be used
+     * for non-transactional reads or queries.
+     *
+     * <p>Note that DirectedReadOptions can only be set for ReadOnlyTransaction and
+     * SingleUseTransaction; it cannot be set for ReadWriteTransaction and
+     * PartitionedDmlTransaction.
```

Comment:
Is this relevant here? This method is not running a transaction.

Reply:
Removed.